tests/unit/TranslateTest.js

```diff
@@ -52,3 +54,38 @@ describe('translate', () => {
         expect(translate.translate('en', ['testKeyGroup', 'testFunction'], {testVariable})).toBe(expectedValue);
     });
 });
+
+describe('Translation Keys', () => {
+    let activeLanguage;
+    let path = '';
+    function matchKeys(source, target, key) {
+        path += key ? `${key}.` : '';
+        const pathLevel = path;
+        if (key && !_.has(target, key)) {
+            console.debug(`🏹 ${path.slice(0, -1)} is missing from ${activeLanguage}.js`);
+            return;
+        }
+        const sourceOBJ = key ? source[key] : source;
+        const targetOBJ = key ? target[key] : target;
+        if (_.isObject(sourceOBJ) && !_.isFunction(sourceOBJ)) {
+            return _.every(_.keys(sourceOBJ), (subKey) => {
+                path = pathLevel;
+                return matchKeys(sourceOBJ, targetOBJ, subKey);
+            });
+        }
+        if (key) {
+            path = path.slice(0, -(key.length - 1));
+        }
+        return true;
+    }
+    it('Does each locale has all the keys', () => {
+        const excludeLanguages = ['en', 'es-ES'];
+        const languages = _.without(_.keys(originalTranslations.default), ...excludeLanguages);
```

Comment:
Can we make this smarter and instead of having to add `es-ES` here make it skip any file that has 2 components? ie: pick up `es` but not `es-ES`

Reply:
That was not the original Idea. After Sometime, we can decide to add more Translations. and Two component locale would be completely valid in that case. So to skip regardless of specification will lead to code change. So I think its better, not to make it smarter. It is sufficient to fulfill its purpose.